src/bunit.web/TestContext.cs

```diff
@@ -42,7 +42,7 @@ public virtual IRenderedComponent<TComponent> RenderComponent<TComponent>(params
 	/// <typeparam name="TComponent">Type of the component to render.</typeparam>
 	/// <param name="parameterBuilder">The ComponentParameterBuilder action to add type safe parameters to pass to the component when it is rendered.</param>
 	/// <returns>The rendered <typeparamref name="TComponent"/>.</returns>
-	public virtual IRenderedComponent<TComponent> RenderComponent<TComponent>(Action<ComponentParameterCollectionBuilder<TComponent>> parameterBuilder)
+	public virtual IRenderedComponent<TComponent> RenderComponent<TComponent>(Action<ComponentParameterCollectionBuilder<TComponent>>? parameterBuilder)
```

Comment:
```suggestion
	public virtual IRenderedComponent<TComponent> RenderComponent<TComponent>(Action<ComponentParameterCollectionBuilder<TComponent>>? parameterBuilder = null)
```

I see your point. Although the overload `RenderComponent<TComponent>(params ComponentParameter[] parameters)` sort of covers the "I don't need to pass any parameters" scenario.

That said, I prefer we make the change such that the `parameterBuilder` is entirely optional.

Reply:
Sorry I missed your point. I thought you were talking for removing the parameter altogether. 

One thing though, Wouldn't this result in an overload ambiguity?

Reply:
Interesting, so no ambiguity at least.
But for this call:

```
RenderComponent<InputText>();
```

Invocation would change to the action overload instead of empty `params` call.